src/Mod/TechDraw/App/DrawViewPart.cpp

```diff
@@ -1498,6 +1498,40 @@ void DrawViewPart::handleChangedPropertyType(Base::XMLReader &reader, const char
     }
 }
 
+// true if owner->element is a cosmetic vertex
+bool DrawUtil::isCosmeticVertex(App::DocumentObject* owner, std::string element)
+{
+    auto ownerView = static_cast<TechDraw::DrawViewPart*>(owner);
+    auto vertexIndex = DrawUtil::getIndexFromName(element);
+    auto vertex = ownerView->getProjVertexByIndex(vertexIndex);
+    if (vertex) {
+        return vertex->getCosmetic();
+    }
+    return false;
+}
+
+// true if owner->element is a cosmetic edge
+bool DrawUtil::isCosmeticEdge(App::DocumentObject* owner, std::string element)
+{
+    auto ownerView = static_cast<TechDraw::DrawViewPart*>(owner);
+    auto edge = ownerView->getEdge(element);
+    if (edge && edge->source() == SourceType::COSMETICEDGE && edge->getCosmetic()) {
+        return true;
+    }
+    return false;
+}
+
+// true if owner->element is a center line
+bool DrawUtil::isCenterLine(App::DocumentObject* owner, std::string element)
```

Comment:
```suggestion
bool DrawViewPart::isCenterLine(App::DocumentObject* owner, std::string element)
```

For those to be DrawViewPart members, we need to change the name qualifier (that marks to which parent function it belongs.

After doing this, you will get a build error. This is because you also need to move the declarations in DrawUtil.h to DrawViewPart.h